spec/spec_helper.rb

```diff
@@ -25,6 +25,13 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  # Randomize specs run order.
+  config.order = :random
+  Kernel.srand config.seed
+
+  # Allow to limit the run of the specs
+  config.filter_run_when_matching :focus
```

Comment:
The attention point for having this enabled is that if someone forgets the focus on the code, then this would be a problem since most of the tests would be skipped.

So I'd like your opinion on this matter

Reply:
I prefer to NOT set this configuration because it's risky and costly to check each PR does not introduce focus method (e.g. `fit`). Could you remove this configuration?